fern/openapi/openapi.json

```diff
@@ -2685,7 +2869,12 @@
             "type": "integer",
             "title": "Optimize Streaming Latency"
           },
-          "model_id": { "type": "string", "title": "Model Id" }
+          "model_id": { "type": "string", "title": "Model Id" },
+          "experimental_input_streaming": {
+            "type": "boolean",
+            "title": "Experimental Input Streaming",
+            "default": false
+          }
```

Comment:
Do we want to double check that this should be exposed?

Reply:
I think this is okay but worth checking with @ajar98 on this one. I believe this is the PR where it was introduced:
https://github.com/vocodedev/vocode-api/commit/becfb3b43b19b55803b2343d9975328cf2fe8293

Reply:
yes this can be exposed